timeflux_dsp/nodes/spectral.py

```diff
@@ -277,7 +303,8 @@ def update(self):
                 tot_power = self.i.data.sum('frequency').values
                 tot_power[tot_power == 0.0] = 1
                 band_power /= tot_power
-
             band['port'].data = pd.DataFrame(columns=self.i.data.space.values, index=self.i.data.time.values,
                                              data=band_power)
+            if self._amplitude:
```

Comment:
Usually, the input signals is a power, so it's already a real values. 
Instead of adding an argument, I'd prefer the user to add a node just after to apply this `abs` operation, insofar as this is really case dependant, and one could also want to take the square, sqrt, the angle, or else...
What is your use-case ?

Reply:
I had a sensible real valued signal (mean zero, sd 1), I passed it into the FFT, I passed that into Bands, I got out a complex number I wasn't expecting and didn't really understand how to interpret. Googling stack exchange suggested I just needed to abs it to get what I wanted, and I figured this was probably the 95% use case for other people.

Reply:
This node may also be used after a Welch node, where you need to average powers in a frequency band. 
For your use-case, I would definitely do a Branch with 3 nodes: FixedWindow(length=2^N) -> FFT -> Abs(on ports)

Reply:
OK, removed